integer/src/bits.rs

```diff
@@ -241,6 +241,11 @@ impl IBig {
             Negative => repr.trailing_ones_neg(),
         }
     }
+
+    #[inline]
+    pub fn count_ones(&self) -> usize {
```

Comment:
Why would you need the functionality to count ones on a signed integer? The implementation here doesn't make sense to me as well...